feconf.py

```diff
@@ -30,8 +31,9 @@
 # Whether we should serve the development or production experience.
 if PLATFORM == 'gae':
     DEV_MODE = (
-        not os.environ.get('SERVER_SOFTWARE')
-        or os.environ['SERVER_SOFTWARE'].startswith('Development'))
+        (not os.environ.get('SERVER_SOFTWARE')
+         or os.environ['SERVER_SOFTWARE'].startswith('Development'))
+        and os.environ.get('MINIFICATION') is None)
```

Comment:
I found the only way to set an environment variable is through the app.yaml like 
```
env_variable:
         NAME: Value
```
only this way will make local variable be visible through ` os.environ`